platform/xbox/crt0.c

```diff
@@ -6,15 +6,42 @@
 
 extern void _PDCLIB_xbox_run_crt_initializers();
 extern int main (int argc, char **argv);
+extern mtx_t _PDCLIB_filelist_mtx;
+extern struct _PDCLIB_file_t * stdin;
+extern struct _PDCLIB_file_t * stdout;
+extern struct _PDCLIB_file_t * stderr;
+
+void _PDCLIB_xbox_run_libc_initializers ()
+{
+    mtx_init(&_PDCLIB_filelist_mtx, mtx_plain);
+    mtx_init(&stdin->mtx, mtx_recursive);
+    mtx_init(&stdout->mtx, mtx_recursive);
+    mtx_init(&stderr->mtx, mtx_recursive);
+}
+
+void _PDCLIB_xbox_run_libc_deinitializers ()
+{
+    mtx_destroy(&_PDCLIB_filelist_mtx);
+    mtx_destroy(&stdin->mtx);
+    mtx_destroy(&stdout->mtx);
+    mtx_destroy(&stderr->mtx);
+}
 
 static int main_wrapper ()
 {
     _PDCLIB_xbox_tss_init();
 
+    _PDCLIB_xbox_run_libc_initializers();
```

Comment:
nit: as this doesn't run a list of callbacks, I feel that `_PDCLIB_xbox_libc_init` is more suitable (closer to the `tss_init`)

Reply:
Agreed and changed.